File: .sdk/tm/php/utility/MakeContext.php
<?php
declare(strict_types=1);

// RailwayStationPhotos SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class RailwayStationPhotosMakeContext
{
    public static function call(array $ctxmap, ?RailwayStationPhotosContext $basectx): RailwayStationPhotosContext
    {
        return new RailwayStationPhotosContext($ctxmap, $basectx);
    }
}

File: .sdk/tm/rb/utility/make_context.rb
# RailwayStationPhotos SDK utility: make_context
require_relative '../core/context'
module RailwayStationPhotosUtilities
  MakeContext = ->(ctxmap, basectx) {
    RailwayStationPhotosContext.new(ctxmap, basectx)
  }
end
